eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Serialisation/Golden/Encoding.hs

```diff
@@ -1135,11 +1136,12 @@ tests =
               ]
        in testCase "ledger state golden test" $
             unless (actual == expected) $
-              assertFailure $
-                unlines
-                  [ "Expected: " ++ show expectedHex
-                  , "Actual: " ++ show actualHex
-                  , diffExpr (CBORBytes expected) (CBORBytes actual)
+              assertFailure . ansiDocToString $
```

Comment:
tasty doesn't play well with color, but we can keep it this way, since we'll need to move this test case to the newer test suite that uses hspec anyways.